tests/test_exceptions.py

```diff
@@ -74,6 +74,13 @@ def test_cross_module_exceptions(msg):
     assert str(excinfo.value) == "'just local'"
 
 
+# TODO: FIXME
+@pytest.mark.xfail(
+    "env.MACOS and env.PYPY",
+    raises=RuntimeError,
+    reason="See Issue #2847, PR #2999, PR #4324",
+    strict=False,
```

Comment:
Could you please append `  # PR #5569` here?

That'll make it a lot easier to backtrack later. (git blaming through e.g. whitespace changes can be a drag.)

Reply:
Actually ... sorry, we should do that only for PyPy, something like
```
    strict=not env.PYPY,  # PR 5569
```

Reply:
```suggestion
    strict=not env.PYPY,  # PR 5569
```